gusto/equations.py

```diff
@@ -484,36 +485,37 @@ def generate_tracer_transport_terms(self, active_tracers):
         return adv_form
 
 
-class ForcedAdvectionEquation(PrognosticEquationSet):
+class CoupledTransportEquation(PrognosticEquationSet):
     u"""
-    Discretises the advection equation with a source/sink term,               \n
+    Discretises a transport equation with a source/sink term,               \n
```

Comment:
I would be tempted to remove the direct reference to source/sink in the comments here, but also to add a comment at the end of the description here saying "This equation can also be enhanced through the addition of sources or sinks by using it with physics schemes."